nuntium/views.py

```diff
@@ -264,6 +264,10 @@ def get_queryset(self):
         qs = qs.filter(writeitinstance=self.writeitinstance).\
             filter(public=True).filter(confirmated=True).\
             filter(author_email=self.message.author_email)
+        if self.message.author_name == '':
+            qs = qs.filter(author_name='')
+        else:
+            qs = qs.exclude(author_name='')
```

Comment:
I would be more comfortable with not allowing messages which are anonymous from being linked to others from the same email address via this view. I can imagine that the clustering of interests of a correspondent might reduce their anonymity. Maybe there's already been some discussion of this though?